styles.css

```diff
@@ -0,0 +1,365 @@
+  flex-direction: column;
+  max-height: 80vh;
+  background-image: url(assets/Img\ Placeholder1.png);
+  background-repeat: no-repeat;
+  background-size: 90%;
+  padding-left: 16px;
+  padding-right: 16px;
+  background-position: top;
+}
+
+.work h2 {
+  text-align: left;
+  margin-left: 36px;
+  margin-top: 24px;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  width: 80%;
+  color: #fff;
+}
+
+.work p {
+  text-align: left;
+  margin-left: 36px;
+  margin-top: 24px;
+  max-width: 80%;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+}
+
+.work .techstack-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  margin-left: 36px;
+  max-width: 80%;
+  margin-top: 10px;
+}
+
+.work li {
+  color: #fff;
+}
+
+.work .project-button:hover {
+  background: rgba(0, 0, 0, 0.3);
+}
+
+.about {
+  background-color: #1c1a19;
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+}
+
+.about .about-me {
+  background-image: url(assets/Frame.png);
+  background-repeat: no-repeat;
+  background-size: 40%;
+  background-position: bottom right;
+  min-height: 80vh;
+  display: flex;
+  flex-direction: column;
+}
+
+.about h2 {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 72px;
+  line-height: 88px;
+  font-family: "Crete Round", sans-serif;
+  margin-top: 50px;
+  color: #ff6b00;
+  margin-left: 20px;
+}
+
+.header .welcome h1 {
+  color: #ff6b00;
+  font-size: 56px;
+  margin-top: 150px;
+  font-family: "Crete Round", sans-serif;
+  margin-left: 10px;
+}
+
+.about p {
+  color: #f4f5f7;
+  margin-left: 20px;
+  max-width: 80%;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 24px;
+}
+
+.about button {
+  background-color: #ff6b00;
+  height: 48px;
+  color: #fff;
+  align-self: flex-start;
+  margin-left: 20px;
+  margin-top: 20px;
+  padding: 12px;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  border: none;
+  font-family: 'Inter', sans-serif;
+}
+
+.about hr {
+  margin-bottom: 10px;
+  width: 90%;
+  align-self: center;
+  color: #fff;
+}
+
+.skills-container {
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  min-height: 100vh;
+  margin-top: 20px;
+}
+
+.skills-container .skills-title {
+  margin-top: 30px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-family: 'InaiMathi', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+}
+
+.skills-container .skills-title h2 {
+  margin-left: 10px;
+}
+
+.skills-container ul {
+  justify-content: space-between;
+  margin-top: 20px;
+}
+
+.skills-container ul li {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 22px;
+  line-height: 43px;
+  color: #fff;
+}
```

Comment:
- [ ] Please 🙏🏽 use the correct styles fonts, colors, margins, for the skills titles: Languages, framework, and skills to be similar to the Figma template 👇🏽 

| Figma template | Your template |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/219785118-3d6d76a4-6724-412e-962f-82c4adfb920a.png) | ![image](https://user-images.githubusercontent.com/58642949/219785270-fb95b3cf-3508-4b6e-986b-d8e257b6e30e.png) |